webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -82,7 +81,7 @@ const TasksTable = ({
     return <span>{__('No Tasks')}</span>;
   }
 
-  const changeTablePage = ({ page, perPage }) => {
+  const changeTablePage = ({ page, per_page: perPage }) => {
```

Comment:
I guess this isn't need anymore as core already take cares of updating the url

Reply:
yup, removed